CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ All notable changes to this project will be documented in this file.
 - chore: team email removed from feedback section in readme
 - chore: updates to npm dependencies
 
+To fully incorporate the fix made to CI/CD pipeline, please redeploy the pipeline stack by following the steps listed on the `main/cicd/README.md` file.
```

Comment:
Probably mention if you use CICD for deploying then redeploy CICD stack?

Reply:
They could be using CI/CD for running integration tests, not necessarily for deployment.

Reply:
So all customers need to redeploy cicd stack regardless? I was running integration tests but never had cicd setup and similarly as part of our GitHub workflow which runs integration tests the CICD role wasn’t being used otherwise tests would have failed. What am I missing?

Reply:
Changing wording as discussed.